core/aws/src/libs/starknet/index.ts

```diff
@@ -1,39 +1,54 @@
 import {
-  Provider, Signer, ec, stark, AddTransactionResponse, GetTransactionStatusResponse,
+  Provider, ec, AddTransactionResponse, GetTransactionStatusResponse, Account, Contract,
 } from 'starknet';
 import BN from 'bn.js';
 
-const { getSelectorFromName } = stark;
-
-// declare type NetworkName = 'mainnet-alpha' | 'georli-alpha';
-
-export enum NetworkName {
-  MAINNET = 'mainnet-alpha',
-  TESTNET = 'georli-alpha'
+const parsedABI = JSON.parse('./abi/terabethia_abi.json');
+
+const NetworkConfig = {
+  testnet: {
+    baseUrl: 'https://alpha4.starknet.io',
+    feederGatewayUrl: 'feeder_gateway',
+    gatewayUrl: 'gateway',
+  },
+  mainnet: {
+    baseUrl: 'https://alpha4.starknet.io', // TBD
+    feederGatewayUrl: 'feeder_gateway', // TBD
+    gatewayUrl: 'gateway', // TBD
+  },
+};
+
+export enum Network {
+  TESTNET = 'testnet',
+  MAINNET = 'mainnet',
 }
```

Comment:
This change breaks import in `[src/functions/sync.check`](https://github.com/Psychedelic/terabethia/blob/master/core/aws/src/functions/sync/check.ts#L5). 

I'd prefer to stick with original `NetworkName`, because [Starknet Provider](https://github.com/0xs34n/starknet.js/blob/3a589f55c6de80b933b2ea8e1ae2444d1964656f/src/provider/sequencer.ts#L49) accepts either `network` or config object. So initiating Provider as `new Provider({ network })` can be just fine (also that's how it's done in `functions/sync/check.ts`. 

Also for mainnet we can probably go with `mainnet-alpha` as a network (instead of TBD).

Reply:
ok, I will rename `Network` 👍 
about the `mainnet-alpha` param, I updated it because I think the guys from starknet changed it in this latest version [provider-doc](https://www.starknetjs.com/docs/api/provider/#creating-an-instance)

Reply:
the doc provides just extended default configuration, you can check the code here: 
https://github.com/0xs34n/starknet.js/blob/develop/src/provider/sequencer.ts#L49

if you specify just `network` then it'll automatically fill these default params (see constructor)